notebooks/enterprise-search/app-search-crawler-to-open-crawler-migration.ipynb

```diff
@@ -0,0 +1,383 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0cccdff9-5ef4-4bc8-a139-6aefa7609f1e",
+   "metadata": {},
+   "source": [
+    "## Hello, future Open Crawler user!\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/elastic/elasticsearch-labs/blob/main/notebooks/enterprise-search/app-search-crawler-to-open-crawler-migration.ipynb)\n",
+    "\n",
+    "This notebook is designed to help you migrate your App Search Web Crawler configurations to Open Crawler-friendly YAML!\n",
```

Comment:
Could we please link App Search Web Crawler to its documentation https://www.elastic.co/guide/en/app-search/current/web-crawler.html? This will help disambiguate between Elastic crawler and App Search crawlers. :)